bin/changelog.txt

```diff
@@ -1,3 +1,25 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  Changelog for:
+    v3.0.7
+
+  * bug fix, + new feature, o other
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Update gs2compiler - fix getangle and getdir
++ Update gs2compiler - added gs2 support for arraylen/sarraylen functions
+* Fixed shoot2 for v6 clients

```

Comment:
```suggestion
+ Added `public function` for serverside scripting, to access functions on an npc from another npc
+ Added ability to get/set tiles on serverside scripting via `level.tiles[index]`
+ Added arrow drop class, so if there's a class named `darts` it will drop this npc instead of the default item. Might change in a future release
* Fixed shoot2 for v6 clients
* Fixed updateboard request causing illegal packet disconnection
* Fixed so player.platform is correctly set when connecting with a v4+ client


```